backend/msgpack/conn_test.go

```diff
@@ -35,65 +40,133 @@ import (
 const (
 	testFakeServerAddr  = "nonexistent"
 	testLocalServerAddr = "127.0.0.1:0"
+
```

Comment:
nit: remove empty line?

Reply:
My thought was to group the variables by "purpose" but I don't have a preference either way.